applications/spark/spark_scripts/configure_spark.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+
+CONFIG_DIR=$(pwd)
+DRIVER_MEMORY_GB=1
+ENABLE_DYNAMIC_ALLOCATION=false
+ENABLE_HISTORY_SERVER=false
+# Many online docs say executors max out with 5 threads.
+EXECUTOR_CORES=5
+PARTITION_MULTIPLIER=1
+SLURM_JOB_IDS=()
+
+function check_errors()
```

Comment:
The functions in this file could benefit from comments/docs that describe what each function does.